chutney/glacio-adapter/src/test/java/integrationtest/EngineIntegrationTest.java

```diff
@@ -19,20 +19,23 @@
 import static com.chutneytesting.engine.api.execution.StatusDto.FAILURE;
```

Comment:
Rather than casting manually to jackson types, use assertj method **asInstanceOf** please.